plugins/modules/acme_certificate.py

```diff
@@ -426,23 +384,23 @@
     challenge: tls-alpn-01
     remaining_days: 60
     data: "{{ sample_com_challenge }}"
-    # We use Let's Encrypt's ACME v2 endpoint
+  # We use Let's Encrypt's ACME v2 endpoint
     acme_directory: https://acme-v02.api.letsencrypt.org/directory
     acme_version: 2
-    # The following makes sure that if a chain with /CN=DST Root CA X3 in its issuer is provided
-    # as an alternative, it will be selected. These are the roots cross-signed by IdenTrust.
-    # As long as Let's Encrypt provides alternate chains with the cross-signed root(s) when
-    # switching to their own ISRG Root X1 root, this will use the chain ending with a cross-signed
-    # root. This chain is more compatible with older TLS clients.
+  # The following makes sure that if a chain with /CN=DST Root CA X3 in its issuer is provided
+  # as an alternative, it will be selected. These are the roots cross-signed by IdenTrust.
+  # As long as Let's Encrypt provides alternate chains with the cross-signed root(s) when
+  # switching to their own ISRG Root X1 root, this will use the chain ending with a cross-signed
+  # root. This chain is more compatible with older TLS clients.
```

Comment:
```suggestion
    # The following makes sure that if a chain with /CN=DST Root CA X3 in its issuer is provided
    # as an alternative, it will be selected. These are the roots cross-signed by IdenTrust.
    # As long as Let's Encrypt provides alternate chains with the cross-signed root(s) when
    # switching to their own ISRG Root X1 root, this will use the chain ending with a cross-signed
    # root. This chain is more compatible with older TLS clients.
```